lib/maths/CTimeSeriesDecompositionDetail.cc

```diff
@@ -255,6 +258,7 @@ const TSizeVecVec SC_TRANSITION_FUNCTION{
     TSizeVec{SC_NORMAL, SC_NORMAL, SC_NORMAL, SC_NORMAL}};
 
 const std::string VERSION_6_3_TAG("6.3");
+const std::string VERSION_6_4_TAG("6.4");
```

Comment:
We probably want to prevent a job that has written 6.4 state from running on a 6.3 node in a mixed 6.3/6.4 cluster.  I guess if this isn't done then a couple of fields won't get initialised.  The way to do that is to change [line 78 of CAnomalyJob.cc](https://github.com/elastic/ml-cpp/blob/d5211a76f9227fb9cbbb44dd10c0126502bc5ef7/lib/api/CAnomalyJob.cc#L78).